plugins/outputs/librato/librato.go

```diff
@@ -36,38 +41,48 @@ var sampleConfig = `
   api_token = "my-secret-token" # required.
   ## Debug
   # debug = false
-  ## Tag Field to populate source attribute (optional)
-  ## This is typically the _hostname_ from which the metric was obtained.
-  source_tag = "host"
   ## Connection timeout.
   # timeout = "5s"
   ## Output Name Template (same as graphite buckets)
   ## see https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md#graphite
-  template = "host.tags.measurement.field"
+  template = "host"
+
+  ## Resolution, in seconds, If the original measurements were reported at a
```

Comment:
can you explain what Resolution does when _writing_ metrics? I couldn't find any documentation for it in their write API